.github/CODEOWNERS

```diff
@@ -0,0 +1,14 @@
+# @node-fetch/Core
+* @bitinn @xxczaki @Richienb @gr2m @jimmywarting
```

Comment:
I'm not sure I understand this line - what is its purpose? Can't it be replaced with just `@node-fetch/core`?

Reply:
@xxczaki Yep. I just realized that. But im not sure if this file supports teams.

Reply:
@NotMoni I think it does, see https://github.com/gatsbyjs/gatsby/blob/master/CODEOWNERS

Reply:
@xxczaki Thanks for that suggestion. It has now been updated.